resources/i18n/en.json

```diff
@@ -1509,5 +1509,7 @@
   "explorer": {
     "ValueRequired": "ValueRequired",
     "InputTooShort": "InputTooShort"
-  }
+  },
+  "NoExistingConnectionExample": "No Connection Example to be copied.",
+  "SSHConnectionExampleClipboardCopy": "SSH Connection Example has been copied to the clipboard."
```

Comment:
It would be better to move these codes into the [`session.applauncher`](https://github.com/lablup/backend.ai-webui/blob/ef0b059ce94115a1f71940f329d14255774d5105/resources/i18n/en.json#L331)

Reply:
I missed it 😢 Now moved to `applauncher`.